examples/example-js-es6/src/index.js

```diff
@@ -91,7 +91,7 @@ const renderWalletList = () => {
         // If the wallet has an eventAction (they should all have an event action...)
         if (eventAction) {
           // Set the name of the wallet into the walletconnect-js state (to use as a reference)
-          walletConnectService.setState({ walletApp: id });
+          walletConnectService.setState({ walletAppId: id });
```

Comment:
Should probably be removed since `setState` is no longer exposed.

Reply:
Great catch, I'll look over this file and update to make sure we can still set the walletAppId, might need a new method on walletConnectService now that setState is removed.

Reply:
Updated to use `walletConnectService.setWalletAppId` method